proto/ibc/core/client/v1/query.proto

```diff
@@ -36,6 +36,11 @@ service Query {
     option (google.api.http).get = "/ibc/core/client/v1/consensus_states/{client_id}";
   }
 
+  // ConsensusStateHeights queries the height of every consensus states associated with a given client.
+  rpc ConsensusStateHeights(QueryConsensusStateHeightsRequest) returns (QueryConsensusStateHeightsResponse) {
+    option (google.api.http).get = "/ibc/core/client/v1/consensus_state_heights/{client_id}";
```

Comment:
I'm pretty indifferent and don't feel too strongly about this but it may appear more standard as suggested below, but perhaps the rest of the team have some thoughts. Feel free to ignore 

```suggestion
    option (google.api.http).get = "/ibc/core/client/v1/consensus_states/heights/{client_id}";
```

Reply:
yeah I think your change make more sense